soc/v2/riscv/common/Kconfig

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+#
+rsource "riscv-privileged/Kconfig"
```

Comment:
let's wait until this riscv-privileged mess is finally moved into arch/, so we do not propagate such abominations.